ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/authentication/AuthenticationUtils.java

```diff
@@ -34,6 +36,15 @@ public static boolean isPermitted(
       final Optional<AuthenticationService> authenticationService,
       final Optional<User> optionalUser,
       final JsonRpcMethod jsonRpcMethod) {
+    return isPermitted(authenticationService, optionalUser, jsonRpcMethod, null);
```

Comment:
should this method still exist? I thought that only the method with the no auth method should exist ....

Reply:
Good point, you made me realise that I didn't implement the auth skipping for web sockets!

Reply:
[d4e4e92](https://github.com/hyperledger/besu/pull/3382/commits/d4e4e9283a0728fdf5e50439d1132bca4dea2aac)